api/tests/test_asset_model_number_api.py

```diff
@@ -0,0 +1,180 @@
+        mock_verify_token.return_value = {'email': self.user.email}
+        response = client.get(
+            f"{self.asset_model_no_url}{self.asset_model_no.id}/",
+            HTTP_AUTHORIZATION="Token {}".format(self.token_user))
+
+        self.assertIn("model_number", response.data.keys())
+        self.assertIn(self.asset_model_no.model_number,
+                      response.data.values())
+        self.assertEqual(response.status_code, 200)
+
+    @patch('api.authentication.auth.verify_id_token')
+    def test_asset_model_number_api_endpoint_cant_allow_put(
+            self, mock_verify_id_token):
+        mock_verify_id_token.return_value = {'email': self.user.email}
+        data = {}
+        response = client.put(
+            self.asset_model_no_url,
+            data=data,
+            HTTP_AUTHORIZATION="Token {}".format(self.token_user))
+        self.assertEqual(response.data, {
+            'detail': 'Method "PUT" not allowed.'
+        })
+        self.assertEqual(response.status_code, 405)
+
+    @patch('api.authentication.auth.verify_id_token')
+    def test_asset_model_number_api_endpoint_cant_allow_patch(
+            self, mock_verify_id_token):
+        mock_verify_id_token.return_value = {'email': self.user.email}
+        data = {}
+        response = client.patch(
+            self.asset_model_no_url,
+            data=data,
+            HTTP_AUTHORIZATION="Token {}".format(self.token_user))
+        self.assertEqual(response.data, {
+            'detail': 'Method "PATCH" not allowed.'
+        })
+        self.assertEqual(response.status_code, 405)
+
+    @patch('api.authentication.auth.verify_id_token')
+    def test_asset_model_number_api_endpoint_cant_allow_delete(
+            self, mock_verify_id_token):
+        mock_verify_id_token.return_value = {'email': self.user.email}
+        data = {}
+        response = client.delete(
+            self.asset_model_no_url,
+            data=data,
+            HTTP_AUTHORIZATION="Token {}".format(self.token_user))
+        self.assertEqual(response.data, {
+            'detail': 'Method "DELETE" not allowed.'
+        })
+        self.assertEqual(response.status_code, 405)
+
+    @patch('api.authentication.auth.verify_id_token')
+    def test_can_post_invalid_payload_model_number(
+            self, mock_verify_token):
+        mock_verify_token.return_value = {'email': self.user.email}
+        data = {
+            "model_number": "",
+            "make_label": self.asset_label.id
+        }
+        response = client.post(
+            self.asset_model_no_url,
+            data=data,
+            HTTP_AUTHORIZATION="Token {}".format(self.token_user))
+        self.assertEquals(response.data, {
+            'model_number': ['This field may not be blank.']})
+        self.assertEqual(response.status_code, 400)
+
+    @patch('api.authentication.auth.verify_id_token')
+    def test_can_post_invalid_payload_make_label(
```

Comment:
For the normal strings (strings without quotes in them) in this PR, refactor them to use single quotes instead of double quotes. It will ensure more consistency across the project.